templates/cli/lib/commands/generic.js.twig

```diff
@@ -122,8 +151,51 @@ const logout = new Command("logout")
             sdk: client
         })
 
-        globalConfig.setCookie("");
-        success()
+        globalConfig.removeLogin(accountId);
+    } catch (e) {
+        error('Unable to log out, removing locally saved session information')
+    }
+    globalConfig.removeLogin(accountId);
+}
+
+const logout = new Command("logout")
+    .description(commandDescriptions['logout'])
+    .configureHelp({
+        helpWidth: process.stdout.columns || 80
+    })
+    .action(actionRunner(async () => {
+        const logins = globalConfig.getLogins();
+        const current = globalConfig.getCurrentLogin();
+
+        if (current === '') {
+            return;
+        }
+        if (logins.length === 1) {
+            await singleLogout(current);
```

Comment:
Also singleLogout is not a very good name. Lets just call it `deleteSession` or `logout`